isaric/parsers/isaric-core.toml

```diff
@@ -1334,7 +1429,9 @@
 [[observation]]
   name = "inability_to_walk_scale"
   phase = "followup"
-  start_date = { field = "flw_survey_date", apply = { function = "startDate", params=[7]}}
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    7,
+  ] } }
```

Comment:
the inability to walk scale questions refer to either today or before covid-19 illness, so params = [7] can be removed if this refers to 'in the last 7 days' for these questions - both the 1-4 scale and the eq5d mobility question